src/linagora.esn.unifiedinbox/i18n/fr.json

```diff
@@ -1,6 +1,6 @@
 {
-  "\"%s\"": "«&nbsp;%s&nbsp;»",
-  "%s items": "%s éléments",
+  "\"%s\"": "\"{{ value }}\"",
```

Comment:
In the old code, there is a « and a » :rofl:

Reply:
I think it was wrong since other languages respect the original version to use `"` and `"`